### ISSUE TITLE
F-Droid

### ISSUE THREAD

**Issue by niqdev** [openwebnet/openwebnet-android#83]
Propose inclusion to the main F-Droid repository
* [Inclusion Policy](https://f-droid.org/docs/Inclusion_Policy)
* [Inclusion How-To](https://f-droid.org/docs/Inclusion_How-To)

**Comment by Poussinou** [comment]
A Request For Packaging is now open [here](https://gitlab.com/fdroid/rfp/issues/624). We are just waiting for your feedback ;)

**Comment by niqdev** [comment]
Thanks and I'm glad you want to include the app on F-Droid!

**Comment by IzzySoft** [comment]
@niqdev the request got stalled on our end as there are non-free dependencies: firebaseui requires parts of GMS. Any chance for a FOSS build variant that comes without that?

**Comment by niqdev** [comment]
Hi @IzzySoft, thanks for the feedback, but unfortunately I don't think that is something that I will be able to do any time soon

**Comment by IzzySoft** [comment]
Hi @niqdev – at least that's not a "no I won't". We'll have to wait then.

I could suggest an "interim solution": if you'd attach the APK to its corresponding release (tag), I could grab it and provide it via [my repo](https://apt.izzysoft.de/fdroid), which acts as something like non-free and staging for F-Droid. Later, when you managed to get rid of the proprietary elements, the app then could move to the official repo (like hundreds of apps did before).

**Comment by niqdev** [comment]
@IzzySoft I had a look at your site now, thanks for your suggestion.

By "drag and drop" the apk when creating a release would be enough? Does it need to be signed? I usually just tag it from cli and never from the ui.

One thing that I always wanted to do, but never had time, was to automate the publish to the Play Store or in this case at least the released apk with Travis when creating a tag. Since you have probably seen lots of projects, would you mind to point me out to some examples of how to do it, if you have any, cos I won't have time to dig into all the docs now, but if there are some example I might just do it quickly.

Thanks

**Comment by IzzySoft** [comment]
> By "drag and drop" the apk when creating a release would be enough? Does it need to be signed?

Yes, and yes. I need to be able to grab the signed `app_release.apk` (not the debug one). Easiest is if it's attached to `releases/` here.

I'm no Android dev myself, so I cannot give advice on Playstore publishing. But maybe Fastlane or Triple-T publisher could help with that (as they do with descriptions and screenshots)? See [here](https://f-droid.org/en/docs/All_About_Descriptions_Graphics_and_Screenshots/) for some details (and links to the two projects).

**Comment by IzzySoft** [comment]
Any news? Or has this project been stopped meanwhile?

**Comment by niqdev** [comment]
Hi @IzzySoft , I'm still maintaining the project, but I'm not actively developing any new feature for now. So if you don't mind, let's keep this issue open just as a reminder.
Thanks

**Comment by IzzySoft** [comment]
Sure @niqdev – I just wanted to avoid "riding dead horses". But if it's still alive, let's give it time to get healthy before we ride it again :rofl: All the best! :crossed_fingers:

**Comment by IzzySoft** [comment]
Well, I'm afraid we'll close [the RFP](https://gitlab.com/fdroid/rfp/-/issues/624). Makes no sense to wait for something that does not happen. Without a FOSS flavor we cannot build anything. Neither could I find Fastlane structures added here.

So no bad feelings; be welcome to ping us once we can pick up again!

**Comment by niqdev** [comment]
No worries @IzzySoft , unfortunately I have no time for this at the moment. I'll reopen the pr if something change!
